zstacklib/zstacklib/test/utils/env.py

```
@@ -2,20 +2,35 @@
+except Exception:
+    import logging
+    logger = logging.getLogger(__name__)
+    jsonobject = None
 
 
 def init_env():
+    if not os.path.exists(envFile):
+        return {}
     with open(envFile, "r") as f:
         try:
             env = yaml.load(f.read(), Loader=yaml.FullLoader)
```

Ruff (0.15.2)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)